testData/solutions/win32.json

```diff
@@ -9,14 +9,248 @@
+                }
+            },
+            "configuration2": {
                 "type": "gpii.settingsHandlers.INISettingsHandler",
                 "options": {
-                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\15.0\\Settings\\VoiceProfiles.ini"
+                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\17.0\\Settings\\VoiceProfiles\\GPII.VPF"
+                },
+                "supportedSettings": {
+                    "Options.PrimarySynthesizer": {},
+                    "ENU-Global.Rate": {},
+                    "ENU-JAWSCursor.Rate": {},
+                    "ENU-Keyboard.Rate": {},
+                    "ENU-MenuAndDialog.Rate": {},
+                    "ENU-Message.Rate": {},
+                    "ENU-PCCursor.Rate": {},
+                    "ENU-Global.Pitch": {},
+                    "ENU-JAWSCursor.Pitch": {},
+                    "ENU-Keyboard.Pitch": {},
+                    "ENU-MenuAndDialog.Pitch": {},
+                    "ENU-Message.Pitch": {},
+                    "ENU-PCCursor.Pitch": {},
+                    "ENU-Global.Volume": {},
+                    "ENU-JAWSCursor.Volume": {},
+                    "ENU-Keyboard.Volume": {},
+                    "ENU-MenuAndDialog.Volume": {},
+                    "ENU-Message.Volume": {},
+                    "ENU-PCCursor.Volume": {},
+                    "ENU-Global.Punctuation": {},
+                    "ENU-JAWSCursor.Punctuation": {},
+                    "ENU-Keyboard.Punctuation": {},
+                    "ENU-MenuAndDialog.Punctuation": {},
+                    "ENU-Message.Punctuation": {},
+                    "ENU-PCCursor.Punctuation": {},
+                    "ENU-Global.SynthLangString": {},
+                    "ENU-JAWSCursor.SynthLangString": {},
+                    "ENU-Keyboard.SynthLangString": {},
+                    "ENU-MenuAndDialog.SynthLangString": {},
+                    "ENU-Message.SynthLangString": {},
+                    "ENU-PCCursor.SynthLangString": {}
                 },
                 "capabilities": [],
                 "capabilitiesTransformations": {
-                    "cloud4allVoiceProfile-GlobalContext\\.Punctuation": {
+                    "Options\\.PrimarySynthesizer": "http://registry\\.gpii\\.net/common/speechSynthesizer",
+                    "ENU-Global\\.Rate": "http://registry\\.gpii\\.net/common/speechRate",
+                    "ENU-JAWSCursor\\.Rate": "http://registry\\.gpii\\.net/common/speechRate",
+                    "ENU-Keyboard\\.Rate": "http://registry\\.gpii\\.net/common/speechRate",
+                    "ENU-MenuAndDialog\\.Rate": "http://registry\\.gpii\\.net/common/speechRate",
+                    "ENU-Message\\.Rate": "http://registry\\.gpii\\.net/common/speechRate",
+                    "ENU-PCCursor\\.Rate": "http://registry\\.gpii\\.net/common/speechRate",
+                    "ENU-Global\\.Pitch":{
```

Comment:
I've found many of these, according to our code style we always put a blank space before a "{". I'm just wondering, is this something that needs to be picked up by our linting rules?